src/System.Numerics.Vectors/src/System/Numerics/Vector4_Intrinsics.cs

```diff
@@ -106,7 +106,8 @@ public void CopyTo(Single[] array)
         public void CopyTo(Single[] array, int index)
         {
             if (array == null)
-                throw new ArgumentNullException("values");
+                // Match the JIT's exception type here. For perf, a NullReference is thrown instead of an ArgumentNull.
```

Comment:
Personal style opinion, but I'd put braces around single-statement bodies that also have comment lines associated with them.